### ISSUE TITLE
Regenerate handshake to fire onremovestream event listener.

### ISSUE THREAD

**Issue by rjatkinson2** [plucky-oatmeal/plucky-oatmeal#53]
After audience members remove their streams from the presenter (turn off their microphone or leave the application altogether), presenterRTC should listen for the change and remove the associated audio tag.

The appropriate way to handle this event is utilizing the onremovestream event listener.  Currently, baseRTC's disconnectFromUser function utilizes webRTC's removeStream method. This alone will not fire the onremovestream event listener. As described in the question below, the handshake must be renegotiated.  Currently, the application is not structured in such a way that this renegotiation can be done easily.

**Comment by ajkitson** [comment]
There is also a close method on the peer connection object, and an associated onclose event. We could try having the audience peerconnection call pc.close() and the presenter peerconnetion remove the peerconnection from the list when it hears the onclose event.

**Comment by rjatkinson2** [comment]
Good point! Both events also lead to onicecandidatestatechange events, but these are delayed a second or two.
